docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -1757,7 +1757,7 @@ a base64-encoded AuthConfig object.
 
         ```
     {
-            "registrytoken": "9cbaf023786cd7..."
+            "identitytoken": "9cbaf023786cd7..."
```

Comment:
I think the change should only be made to v1.23 and up, since identitytoken did not exist in 1.22.